docs/design/hub-and-spoke/switch-validate/switch-validate.md

```diff
@@ -5,9 +5,680 @@
+  "cookie_value": "LLDP_INPUT_PRE_DROP", // not implemented yet
+  "table_id": 10,
+  "priority": 5,
+  "flow_id": "9",
+  "flow_path": "10a1df3a-dc30-453e-8671-246f82ea0e77",
+  "y_flow_id": "111",
+  "flags": [
+    "RESET_COUNTERS"
+  ],
+  "match": [
+    {
+      "value": 1,
+      "mask": 1,
+      "field": "IN_PORT"
+    }
+  ],
+  "instructions": {
+    "go_to_table": 30,
+    "go_to_meter": 40,
+    "write_metadata": {
+      "value": 200,
+      "mask": 200
+    },
+    "apply_actions": [
+      "SET_FIELD"
+    ],
+    "write_actions": [
+      "SET_FIELD"
+    ]
+  }
+}
+```
+
+</details>
+
+### Validate rules, meters, lags and groups installed on the switch v2
+
+`GET https://{host}/v2/switches/{switch_id}/validate`
+
+**By Default:**
+
+* `flow info` is set of values: for groups - `flow_id, flow_path`, for meters and rules - `flow_id, flow_path, y_flow_id`.
+* without any query params response contains v2 info about groups, lags, meters and rules (with flow_info).
+* groups[excess], meters[excess] and rule[excess] do not contain flow info.
+
+Response payload
+
+```json
+{
+  "as_expected": true,
+  "groups": {
+    "as_expected": true,
+    "excess": [
+      <v2GroupInfo> // without flow_id and flow_path
+    ],
+    "missing": [
+      <v2GroupInfo>
+    ],
+    "proper": [
+      <v2GroupInfo>
+    ],
+    "misconfigured": [
+      {
+        "id": "12123213",
+        "expected": {
+          <v2GroupInfo>
+        },
+        "discrepancies": {
+          <v2GroupInfo>
+        }
+      }
+    ]
+  },
+  "logical_ports": {
+    "error": "some_error",
+    "as_expected": true,
+    "excess": [
+      <v2LogicalPortInfo>
+    ],
+    "missing": [
+      <v2LogicalPortInfo>
+    ],
+    "proper": [
+      <v2LogicalPortInfo>
+    ],
+    "misconfigured": [
+      {
+        "id": "123123",
+        "expected": {
+          <v2LogicalPortInfo>
+        },
+        "discrepancies": {
+          <v2LogicalPortInfo>
+        }
+      }
+    ]
+  },
+  "meters": {
+    "as_expected": true,
+    "excess": [
+      <v2MeterInfo> // without flow_id, y_flow_id and flow_path
+    ],
+    "missing": [
+      <v2MeterInfo>
+    ],
+    "proper": [
+      <v2MeterInfo>
+    ],
+    "misconfigured": [
+      {
+        "id": "1232131",
+        "expected": {
+          <v2MeterInfo>
+        },
+        "discrepancies": {
+          <v2MeterInfo>
+        }
+      }
+    ]
+  },
+  "rules": {
+    "as_expected": true,
+    "excess": [
+      <v2RuleInfo> // without flow_id, y_flow_id and flow_path
+    ],
+    "missing": [
+      <v2RuleInfo>
+    ],
+    "proper": [
+      <v2RuleInfo>
+    ],
+    "misconfigured": [
+      {
+        "id": "asdsad",
+        "expected": {
+          <v2RuleInfo>
+        },
+        "discrepancies": {
+          <v2RuleInfo>
+        }
+      }
+    ]
+  }
+}
+```
+
+### Validate rules, meters, lags and groups installed on the switch with filters
+
+### Including specific info in response
```

Comment:
Including ==> Requesting?

i.e: Requesting validation of specific switch objects/subjects.